src/core/xfa/template.js

```diff
@@ -4649,7 +4641,8 @@ class Template extends XFAObject {
       const flush = index => {
         const html = root[$flushHTML]();
         if (html) {
-          hasSomething = true;
+          hasSomething =
+            hasSomething || (html.children && html.children.length !== 0);
```

Comment:
Nit: Here, and similarly below, this can be a bit more compact by instead formatting it as:
```javascript
hasSomething ||= (html.children && html.children.length !== 0);
```

Reply:
I wrote that but the linter failed. And I tried again with parenthesis and it failed again. Do I miss something ?